pkg/sql/rowexec/columnbackfiller.go

```diff
@@ -22,13 +22,28 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/hlc"
 )
 
+const (
+	// maxCommitWaitFns is the maximum number of commit-wait functions that the
+	// columnBackfiller will accumulate before consuming them to reclaim memory.
+	// Each function retains a reference to its corresponding TxnCoordSender, so
+	// we need to be careful not to accumulate an unbounded number of these
+	// functions.
+	maxCommitWaitFns = 128
```

Comment:
not-even-nit: I have a knee-jerk reaction to consts vs cluster settings just for how many times I've had something OOMing or whatever and wished we had a setting to get us out of a bind, but this seems fine unless you think we might want to tune it up/down.